sydent/db/invite_tokens.py

```diff
@@ -20,16 +20,48 @@ class JoinTokenStore(object):
     def __init__(self, sydent):
         self.sydent = sydent
 
-    def storeToken(self, medium, address, roomId, sender, token):
+    def storeToken(self, medium, address, roomId, sender, token, originServer=None, originId=None, commit=True):
+        """Stores an invite token.
+        
+        :param medium: The medium of the token.
+        :type medium: str
+        :param address: The address of the token.
+        :type address: str
+        :param roomId: The room ID this token is tied to.
+        :type roomId: str
+        :param sender: The sender of the invite.
+        :type sender: str
+        :param token: The token itself.
+        :type token: str
+        :param originServer: The server this invite originated from (if
+            coming from replication).
+        :type originServer: str, None
```

Comment:
```suggestion
        :type originServer: str|None
```

unless you have reason to believe your way is correct?

likewise everywhere else...